src/hotspot/share/runtime/globals.hpp

```diff
@@ -443,7 +443,7 @@ const int ObjectAlignmentInBytes = 8;
   product(bool, LogEvents, true, DIAGNOSTIC,                                \
           "Enable the various ring buffer event logs")                      \
                                                                             \
-  product(uintx, LogEventsBufferEntries, 20, DIAGNOSTIC,                    \
+  product(int, LogEventsBufferEntries, 20, DIAGNOSTIC,                      \
```

Comment:
I was going to say this should keep its unsignedness but it is treated as an `int` everywhere anyway.